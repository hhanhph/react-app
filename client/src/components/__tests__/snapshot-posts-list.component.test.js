import React from 'react';
import renderer from 'react-test-renderer';

import PostsList from '../posts-list.component';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<PostsList />).toJSON();
  expect(tree).toMatchSnapshot();
});