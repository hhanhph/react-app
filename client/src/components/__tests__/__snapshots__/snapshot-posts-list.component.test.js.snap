// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <h3>
    News Feed
  </h3>
  <div
    className="row pt-5"
  >
    <div
      className="col-12 col-lg-6 offset-lg-3"
    />
  </div>
</div>
`;
